saleor_app_framework/app_framework/core/install.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import secrets
+import string
+from typing import Awaitable, Callable, List
+
+from ..core.conf import settings
+from .errors import InstallAppError
+from .graphql import GraphqlError, get_executor, get_saleor_api_url
+from .types import AppToken, DomainName, Url, WebhookData
+
+logger = logging.getLogger(__file__)
+
+
+CREATE_WEBHOOK = """
```

Comment:
Loose though - maybe it would make sense to keep queries in separate files for each module? For example `app_framework/core/queries.py`? Just an idea, maybe it's too early for something like that.